MetadataExtractor/Formats/Apple/BplistReader.cs

```diff
@@ -123,7 +108,7 @@ static object HandleInt(ref BufferReader reader, byte marker)
 
         static Dictionary<byte, byte> HandleDict(ref BufferReader reader, byte count)
         {
-            var keyRefs = ArrayPool<byte>.Shared.Rent(count);
+            Span<byte> keyRefs = stackalloc byte[count]; // count is < 256
 
             for (int j = 0; j < count; j++)
             {
```

Comment:
This bit can be changed from:

```c#
for (int j = 0; j < count; j++)
{
    keyRefs[j] = reader.GetByte();
}
```

To

```c#
reader.GetBytes(keyRefs);
```

Reply:
We're relying on an implicit upcast from byte to int here -- so we can't read directly. However, let me refactor to avoid the extra calls / and bounds checks.